weheat/abstractions/heat_pump.py

```diff
@@ -1,12 +1,17 @@
+"""Weheat heat pump abstraction from the API."""
 from enum import Enum, auto
 from weheat.configuration import Configuration
 from weheat.api_client import ApiClient
 from weheat.api.heat_pump_log_api import HeatPumpLogApi
+from weheat.api.energy_log_api import EnergyLogApi
+from datetime import datetime, timedelta
+
+START_DATE = datetime(2024, 1, 1, 0, 0, 0)
```

Comment:
are you going to update the package in 2025? or what the goal of this hard coded value? Pretty sure first heat pump logs start somewhere in April 2023.

Reply:
changed to 2023 and no need to update it since it uses the date range from 2023 to now to sum all energy usage.